projects/kit/components/stepper/stepper.component.ts

```diff
@@ -68,7 +62,12 @@ export class TuiStepperComponent {
 
     constructor(
         @Inject(ChangeDetectorRef) private readonly changeDetectorRef: ChangeDetectorRef,
-    ) {}
+        @Inject(ElementRef) private readonly elementRef: ElementRef<HTMLElement>,
+        @Inject(TuiScrollService) private readonly scrollService: TuiScrollService,
+        @Inject(TuiResizeService) resize$: Observable<void>,
+    ) {
+        resize$.subscribe(() => this.scrollIntoView(this.activeItemIndex));
```

Comment:
I'm not sure, but for safety takeUntil is needed?

Reply:
there is takeUntil inside resizeService